src/util.spec.ts

```diff
@@ -22,6 +22,7 @@ describe('Util', () => {
             expect(util.isAssignableExpression('function test(): endFunction')).to.be.false;
             expect(util.isAssignableExpression('if true then print true')).to.be.false;
             expect(util.isAssignableExpression('while true: print true')).to.be.false;
+            expect(util.isAssignableExpression('a.b.c = "test"')).to.be.false;
```

Comment:
We need to test the indexed set as well. 
```suggestion
            expect(util.isAssignableExpression('a.b.c = "test"')).to.be.false;
            expect(util.isAssignableExpression('a["b"]["c"] = "test"')).to.be.false;
```

Reply:
> We need to test the indexed set as well.

Alright, creating that now.